src/peer_digest.cc

```diff
@@ -332,7 +332,7 @@ peerDigestRequest(PeerDigest * pd)
     /* update timestamps */
     pd->times.requested = squid_curtime;
     pd_last_req_time = squid_curtime;
-    req->flags.cachable = true;
+    req->flags.missCachingDecision = true;
```

Comment:
Setting this flag true is always a risky operation worth documenting.
```suggestion
    // XXX flag currently defaults to false. It should be true to allow collapsing and re-sharing of digests while they are fresh.
    req->flags.missCachingDecision = true;
```